packages/docs/examples/va-progress-bar/Sizing.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <div class="flex lg6 xs12">
+    <div class="my-2 text--bold muted">Size "small":</div>
```

Comment:
Here we probably want to just write 'small', '25px' etc. Let's leave to user figuring out what the example is about.